dvc/command/add.py

```diff
@@ -1,60 +0,0 @@
-import os
-
-from dvc.command.common.base import CmdBase
-from dvc.logger import Logger
-from dvc.state_file import StateFile
-from dvc.path.data_item import DataItem
-
-
-class CmdAdd(CmdBase):
```

Comment:
Are you sure that `dvc add` is not needed. How about `dvc-git` scenarios without repro and staging files?

Reply:
I agree that we need add/remove. I returned it in the upcoming patch, will update pull-request soon.